src/core/Directus/Util/Installation/stubs/config.stub

```diff
@@ -8,7 +8,7 @@ return [
 
     'settings' => [
         'logger' => [
-            'path' => __DIR__ . '/../logs',
+            'path' => '{{logs_path}}',
```

Comment:
I think this might evaluate to `'__DIR__ . '/../logs''`

Reply:
No! Here we'll have a dynamic variable.` '__DIR__ . '/../logs'` this can be considered as a default value.

Reply:
Oh right, it evaluates to the the result of __DIR__. Let's make sure __DIR__ is the right folder and we're all set!